src/simtools/simtel/simtel_config_writer.py

```diff
@@ -249,15 +283,16 @@ def _write_random_seeds_file(self, sim_telarray_seeds, config_file_directory):
         random_instances_of_instrument: int
             Number of random instances of the instrument.
         """
-        random.seed(sim_telarray_seeds["seed"])
         self._logger.info(
             "Writing random seed file "
             f"{config_file_directory}/{sim_telarray_seeds['seed_file_name']}"
             f" (global seed {sim_telarray_seeds['seed']})"
         )
-        random_integers = [
-            random.randint(0, 2**32 - 1) for _ in range(sim_telarray_seeds["random_instances"])
-        ]
+        if sim_telarray_seeds["random_instances"] > 1024:
```

Comment:
Not sure why you decided specifically on 1024 instead of a round number like a 1000, but OK.

Reply:
Ahh OK, that's fine then.